### PR TITLE
#215 Add extension

### PR DESCRIPTION
**Pull request by kaklakariada**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (tkilias) [inline .vscode/settings.json:1]: are you sure you wanted commit this file?

File: README.md [hunk 2]
```diff
@@ -39,7 +39,8 @@ For more information please check out the following guides.
 
 ## Information for Contributors
 
-* [Developer Guide][developer-guide]
+* [General Developer Guide for Import-Export UDF][developer-guide]
```
Comment (ckunki) [inline README.md:42]: I could not find file "developer_guide":

File: doc/developers_guide/developers_guide.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Developers Guide
+
+This guide contains information for developers.
+
+## Working With the Extension
```
Comment (ckunki) [inline doc/developers_guide/developers_guide.md:5]: Not sure whether the term "Extension" is self-explaining here. 
Should we add an introduction passage and/or a link?
How about a term like "Managed Extension" or "Plugin" or "Integration with Exasol and Exasol SAAS"?

Comment (ckunki) [reply doc/developers_guide/developers_guide.md:5]: Potentially also the descriptions for "using a local, non-published..." could turn out to be generic and if so, may be moved to the extension manager developers guide or another central place.

File: extension/generate-config.sh [hunk 4]
```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```
Comment (ckunki) [inline extension/generate-config.sh:1]: Please excuse if I might be somewhat allergic to the term "config". 
Could this script and its name be another hint that there is something like a "plugin" / "integration" / "adapter" or "description" required in order to enable the "cloud-storage-extension" to be managed with the extension manager? 
If the answer is yes, then maybe it's worth to coin a phrase for this piece of software/config etc.

We also should describe the life cycle of this config: 
- where does it come from, when / triggered by what is it generated, by whom?
- what is the syntax or container format? (apparently typescript) 
- what is the content? (apparently file name, size and version of the jar)
- who is the receiver and what are the use cases?

File: .vscode/settings.json (hunk 1 above)
Comment (kaklakariada) [reply .vscode/settings.json:1]: Yes. This helps configuring formatter etc. for vscode.

File: README.md (hunk 2 above)
Comment (kaklakariada) [reply README.md:42]: It is at the end of the file:
```
[developer-guide]: https://github.com/exasol/import-export-udf-common-scala/blob/master/doc/development/developer_guide.md
```

File: doc/developers_guide/developers_guide.md (hunk 3 above)
Comment (kaklakariada) [reply doc/developers_guide/developers_guide.md:5]: Added a link to the extension manager

File: extension/generate-config.sh (hunk 4 above)
Comment (kaklakariada) [reply extension/generate-config.sh]: renamed to description

**Review by ckunki — DISMISSED** [review]

**Pull request merged**